scraper/__init__.py

```diff
@@ -0,0 +1,82 @@
+from scrapy.crawler import CrawlerProcess
+
+import recipescrapers.recipe_scrapers
+import certifi
+from elasticsearch import Elasticsearch
+from dotenv import load_dotenv
+import os
+import math
+
+from .allrecipes import AllRecipes
+from .bbcfood import BBCFood
+from .bbcgoodfood import BBCGoodFood
+from .bettycrocker import BettyCrocker
+from .bonappetit import BonAppetit
+from .budgetbytes import BudgetBytes
+# from .closetcooking import ClosetCooking
+from .cookstr import Cookstr
+from .copykat import CopyKat
+from .delish import Delish
+# from .epicurious import Epicurious
+# from .food import Food
+# from .foodnetwork import FoodNetwork
+# from .foodrepublic import FoodRepublic
+# from .tasty import Tasty
+
+
+HEADERS = {
+    'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64; rv:48.0) Gecko/20100101 Firefox/48.0'
+}
+
+URLS = {
+    # recipescrapers.recipe_scrapers.BBCFood.host(): BBCFood,
+    # recipescrapers.recipe_scrapers.BBCGoodFood.host(): BBCGoodFood,
+    # recipescrapers.recipe_scrapers.BettyCrocker.host(): BettyCrocker,
+    # recipescrapers.recipe_scrapers.BonAppetit.host(): BonAppetit,
+    # recipescrapers.recipe_scrapers.AllRecipes.host(): AllRecipes,
+    # recipescrapers.recipe_scrapers.BudgetBytes.host(): BudgetBytes
+    # recipescrapers.ClosetCooking.host(): ClosetCooking,
+    # recipescrapers.recipe_scrapers.Cookstr.host(): Cookstr,
+    # recipescrapers.recipe_scrapers.CopyKat.host(): CopyKat,
+    recipescrapers.recipe_scrapers.Delish.host(): Delish,
+    # recipescrapers.Epicurious.host(): Epicurious,
+    # recipescrapers.Food.host(): Food,
+    # recipescrapers.FoodNetwork.host(): FoodNetwork,
+    # recipescrapers.FoodRepublic.host(): FoodRepublic,
+    # recipescrapers.Tasty.host(): Tasty
+}
+
+
+def connect_to_es():
+    load_dotenv(dotenv_path='Recipes/api/.env')
```

Comment:
I would move env vars loading to the beginning of your main, you might have more vars in your code so why not load everything from the start.